backends/README.md

```diff
@@ -0,0 +1,49 @@
+# All Backends in VTL Engine
+
+VTL Engine supports the following backends:
+   - Cuda
+   - OpenCL
+   - OpenMP
+   - Vulkan
+
+## Cuda
+ First install cuda toolkit: https://developer.nvidia.com/cuda-downloads
+  ### usage:
+  ```vlang
+        import vtl
+        vtl_tensor := vtl.from_array([1.0, 2, 3, 4], [2, 2])!
+        vtl_tensor.cuda()
+        println('My device: ${vtl_tensor.device}')
+
+  ```
+
+## OpenCL
+  First install opencl toolkit: https://www.khronos.org/opencl/
+  ### usage:
+  ```vlang
+        import vtl
+        vtl_tensor := vtl.from_array([1.0, 2, 3, 4], [2, 2])!
+        vtl_tensor.opencl()
```

Comment:
no need to have an extra backend for opencl. We must use VCL since it is the V official OpenCL wrapper

https://github.com/vlang/vtl/blob/ad0161891a4795fa99274d795cc2089c5c41743c/src/tensor_vcl_d_vcl.v#L17

we could have an alias function called opencl tho